iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergingNetworkListener.java

```diff
@@ -35,8 +35,24 @@ public void onRemoval(final Identifiable identifiable) {
         // Not implemented yet !
     }
 
-    @Override
     public void onUpdate(final Identifiable identifiable, final String attribute, final Object oldValue, final Object newValue) {
         // Not implemented yet !
     }
+
+    @Override
+    public void onUpdate(Identifiable identifiable, String attribute, String variantId, Object oldValue, Object newValue) {
+        Objects.requireNonNull(identifiable, "identifiable is null");
+        if (identifiable instanceof DanglingLine) {
+            final DanglingLine dl = (DanglingLine) identifiable;
+            final String ucteCode = dl.getUcteXnodeCode();
+            final MergedLine mergedLine = index.getMergedLineByCode(ucteCode);
+            if (mergedLine != null) {
+                if (attribute.contains("p")) {
+                    mergedLine.computeAndSetP0();
+                } else if (attribute.contains("q")) {
+                    mergedLine.computeAndSetQ0();
+                }
+            }
+        }
+    }
```

Comment:
I thought we discussed about that: is it OK for you to move this code to a new dedicated listener (DanglingLinePowerListener) and attach it in the constructor of the MergingView?